ui/src/shared/components/CustomTimeIndicator.tsx

```diff
@@ -2,17 +2,21 @@ import React, {SFC} from 'react'
 import _ from 'lodash'
 
 import {TEMP_VAR_DASHBOARD_TIME} from 'src/shared/constants'
+import {QueryConfig} from 'src/types/query'
 
 interface Query {
-  query: string
+  config: QueryConfig
+  text: string
 }
 
 interface Props {
   queries: Query[]
```

Comment:
is this `Query` different from `QueryConfig`?

Reply:
Yes